packages/devtools_app/lib/src/split.dart

```diff
@@ -292,17 +292,23 @@ class _SplitState extends State<Split> {
 class DefaultSplitter extends StatelessWidget {
   const DefaultSplitter({@required this.isHorizontal});
 
-  static const double size = 24.0;
+  static const double iconSize = 24.0;
+  static const double splitterWidth = 12.0;
 
   final bool isHorizontal;
 
   @override
   Widget build(BuildContext context) {
     return Transform.rotate(
       angle: isHorizontal ? degToRad(90.0) : degToRad(0.0),
-      child: const Icon(
-        Icons.drag_handle,
-        size: size,
+      child: Align(
```

Comment:
@pq here is a simple flutter layout lint. Lint if Align is used in a case where `Center` would have the same effect and quick fix to using Center.

Reply:
https://github.com/dart-lang/linter/issues/2157

Reply:
Thanks!